pipelines/manual-release-pipeline.yml

```diff
@@ -781,13 +783,46 @@ jobs:
     - get: census-rm-kubernetes-release
     - get: census-rm-deploy
     - task: "Run Helm"
+      file: census-rm-deploy/tasks/helm.yml
+      params:
+        ADMIN_SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        ENV: ((gcp-environment-name))
+        KUBERNETES_CLUSTER: rm-k8s-cluster
+        RABBITMQ_CONFIG_VALUES_FILE: ((rabbit-config))
+      input_mapping: {census-rm-kubernetes-dependencies-repo: census-rm-kubernetes-release}
+
+- name: "Deploy Monitoring"
+  disable_manual_trigger: true
+  serial: true
+  serial_groups: [action-scheduler,
+                  action-worker,
+                  case-api,
+                  case-processor,
+                  fieldwork-adapter,
+                  notify-processor,
+                  uac-qid-service,
+                  pubsubsvc,
+                  print-file-service,
+                  exception-manager,
+                  toolbox,
+                  rabbitmonitor]
+  on_error: *slack_error_alert
+  on_failure: *slack_failure_alert
+  plan:
+    - get: every-minute
+      trigger: true
+      passed: ["Run Helm"]
+    - get: census-rm-kubernetes-release
+    - get: census-rm-deploy
+    - task: "Deploy Monitoring"
```

Comment:
👇 This is still running the helm task. Shouldn't it be changed to run the monitoring task?

Reply:
Good spot!